content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md

```diff
@@ -17,6 +17,36 @@ weight: 1
 
 # Corda Enterprise release notes
 
+## Corda Enterprise 4.8.8
+
+Corda Enterprise 4.8.8 is a patch release of Corda Enterprise focused on CorDapp response time and security improvements.
+
+### Upgrade recommendation
+
+As a developer, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and you can find the latest upgrade guide [here](../../../../../en/platform/corda/4.9/enterprise/upgrading-index.md).
+
+As a node operator, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html).
+
+### Fixed issues
+
+In this patch release:
+* Java serialization has been disabled in the Corda firewall, closing a potential security vulnerability.
+* The time it takes for first-time flows to be run in a signed CorDapp has been significantly reduced by automatically closing any `AttachmentsClassLoader` that is evicted from the cache.
```

Comment:
This is true, _but_ I suspect you'd need to be a _complete_ Corda Nerd to understand the cause-vs-effect relationship between slow first-time flows and the `AttachmentsClassLoader` cache :wink:. Specifically:
- We introduced caching for `AttachmentsClassLoader`, which slowly leaked file descriptors.
- We fixed this leak one way, but accidentally made first-time flows very slow too.
- Now we're fixing the leak differently, which restores the performance for first-time flows.

I'll defer to @adelel1's judgement here, but I suspect that it's enough just to say that we're fixing a regression introduced in Corda 4.8.thingy by CORDA-4120.

Reply:
We do need to say that the speed up was due to a regression, which has been resolved. So people don't expect all versions to be faster in this release. Something like the following....

"The time it takes for first-time flows to be run in a signed CorDapp has been significantly reduced. This was a regression introduced by the fix for CORDA-4120 in 4.8. Regression has now been fixed."